### PR TITLE
Version change

### PR DESCRIPTION
**Pull request by defei-coder**

### CODE REVIEW

File: installer/centos7.5/SPECS/mluops-independent.spec [hunk 1]
```diff
@@ -76,6 +76,8 @@ cp $RPM_SOURCE_DIR/neuware-env.conf $RPM_BUILD_ROOT/etc/ld.so.conf.d/
 %postun -p /sbin/ldconfig
 
 %changelog
+* Fri Dec 12 2022 Cambricon Software Team <service@cambricon.com>
```
[the remark this answers is not in the document]
Comment (defei-coder) [reply installer/centos7.5/SPECS/mluops-independent.spec]: 已修改

**Review by WangYuan-cn — APPROVED** [review]

File: docs/bangc-docs/api_guide/update.rst [hunk 2]
```diff
@@ -4,6 +4,22 @@ Update History
...
 
+* V0.4.0
+
+  **Date:** December 12, 2022
+
+  **Changes:**
+
+  - Supported boards and operators can be optional while building mlu-ops.
+  - Supported MLU590.
+  - Supported MLU operators performance comparison.
```
Comment (GraceGuanH) [inline docs/bangc-docs/api_guide/update.rst:15]: Supported MLU operation performance comparison.

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 3]
```diff
@@ -44,6 +50,36 @@
...
+v0.4.0
+-----------------
+
+特性变更
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  编译支持板卡、算子可选。
+-  支持MLU590。
+-  支持MLU算子性能比对功能。
+- 新增支持以下算子：
```
Comment (GraceGuanH) [inline docs/bangc-docs/release_notes/bangc_ops.rst:62]: 新增支持以下算子：这一行和上面对齐

**Review by GraceGuanH — COMMENTED** [review]
Please check the comments.

**Pull request merged**